source/server/getting-started/configuration/sponge-conf.rst

```diff
@@ -254,10 +254,10 @@ was generated using SpongeForge build 1399, SpongeAPI version 4.1:
             chunk-gc-load-threshold=0
             # The tick interval used to cleanup all inactive chunks in a world. 
             # Set to 0 to disable which restores vanilla handling. (Default: 1)
```

Comment:
Been a while since I updated sponge, but last I checked there's a config option that you have to turn on to populate the entities list. Otherwise, it defaults to the group's setting, i.e. ambient, aquatic, etc.